build/ci-tf-bin-arm64.Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM --platform=linux/arm64 registry.access.redhat.com/ubi8/ubi-minimal:latest AS builder
```

Comment:
we really need two duplicated dockerfiles?
the base image ``registry.access.redhat.com/ubi8/ubi-minimal:latest`` is multi-arch

Reply:
Did not know that. I'm working on a unified Dockerfile then.

Thanks.